resources/definitions/fdmprinter.def.json

```diff
@@ -3529,6 +3529,19 @@
                     "enabled": "retraction_hop_enabled and extruders_enabled_count > 1",
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
+                },
+                "retraction_hop_after_extruder_switch_height": 
+                {
+                    "label": "Z Hop After Extruder Switch Height",
+                    "description": "The height difference when performing a Z Hop after extruder switch.",
+                    "unit": "mm",
+                    "type": "float",
+                    "default_value": 1,
```

Comment:
I see you have copied the `default_value` from `retraction_hop`, to not get different behaviour if people leave the settings on default. This is fine, but consider that (the config files of) some types of printer (like the ultimaker3 or s5 for instance) can override the previous setting, and this one will not change. In that case, it will behave differently.

Luckily, this is easy to fix; in addition to `"default_value": 1,`, also add `"value": "retraction_hop",`. This will ensure that it copies the value of `retraction_hop` to here (even if overridden or changed by the user), unless this value (`retraction_hop_after_extruder_switch_height`) is changed manually by the user.

Reply:
Thank you, added `"value": "retraction_hop"`